src/test/scala/kamon/logback/LogbackSpanConverterSpec.scala

```diff
@@ -49,6 +50,40 @@ class LogbackSpanConverterSpec extends WordSpec with Matchers with Eventually {
         memoryAppender.getLastLine shouldBe traceID.string
       }
 
+      "report the its context in MDC" in {
```

Comment:
These 2 are basically the same test, right? it would be nice to have assertions after the `withContext` to ensure that the trace and span keys are no longer available on the MDC.

Reply:
more or less same tests, which could be merged, but for clarity i'd like to have them separated. will put assertions